client/client.go

```diff
@@ -17,58 +17,40 @@ package client // import "go.opentelemetry.io/collector/client"
 
 import (
 	"context"
-	"net"
-	"net/http"
-
-	"google.golang.org/grpc/peer"
 )
 
 type ctxKey struct{}
 
 // Client represents a generic client that sends data to any receiver supported by the OT receiver
 type Client struct {
-	IP string
+	IP   string
```

Comment:
I think this should be "net.Addr" to not do the parsing unnecessary for gRPC for example. Also it is more generic. Probably separate issue/PR but I saw it now.